dashboard/config/bits-kcsan.config

```diff
@@ -0,0 +1,47 @@
+# Additional config switches for KCSAN kernels.
+# To be used by kcsan_from_kasan_config.sh
+
+# Disable KASAN:
+# CONFIG_KASAN is not set
+CONFIG_VMAP_STACK=y
+
+# Enable KCSAN:
+CONFIG_KCSAN=y
+# CONFIG_KCSAN_EARLY_ENABLE is not set
+CONFIG_KCSAN_UDELAY_MAX_TASK=80
+CONFIG_KCSAN_UDELAY_MAX_INTERRUPT=20
+CONFIG_KCSAN_DELAY_RANDOMIZE=y
+CONFIG_KCSAN_WATCH_SKIP_INST=2000
+# Conservative data-race detection for now.
+# CONFIG_KCSAN_REPORT_RACE_UNKNOWN_ORIGIN is not set
+CONFIG_KCSAN_IGNORE_ATOMICS=y
+# CONFIG_KCSAN_PLAIN_WRITE_PRETEND_ONCE is not set
+# CONFIG_KCSAN_DEBUG is not set
+
+# Avoid reboot loop in instrumented kernel:
+# CONFIG_PREEMPT is not set
+CONFIG_PREEMPT_NONE=y
+# CONFIG_PARAVIRT is not set
```

Comment:
We don't need this whole block. This is just a work around some KMSAN bugs.
CONFIG_PARAVIRT should work and it useful. syzbot is PARAVIRT.